compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -260,6 +265,218 @@ static CodeGenPipeline getTensorCorePipeline(Type elementType) {
   };
   return codegenPipeline;
 }
+//====---------------------------------------------------------------------===//
+// Vector Distribution Reduction Pipeline Configuration
+//====---------------------------------------------------------------------===//
+
+/// Check if `op` is a linalg.reduce or a linalg.generic that has at least one
+/// reduction iterator.
+static bool hasReductionIterator(linalg::LinalgOp &op) {
+  return isa<linalg::ReduceOp>(op) ||
+         (isa<linalg::GenericOp>(op) &&
+          llvm::any_of(op.getIteratorTypesArray(),
+                       linalg::isReductionIterator));
+}
+
+// Get the bitwidth of the operation.
+static llvm::FailureOr<int64_t> getBitWidth(linalg::LinalgOp op) {
```

Comment:
rename to `getReductionBitWidth`

Reply:
Any reasons why? It doesn't care whether it's a reduction or an elementwise operation.